File: 1-Iniciante/Ex1008.java
import java.util.Scanner;

public class Ex1008 { //para ser aceito no URI mude o nome da classe para Main
    public static void main(String args[]) {
        /* variavies */
        Scanner scanner = new Scanner(System.in);
        int n, horasTrabalhadas;
        Double valorHoraTrab, salario;

        /* entrada */
        n = scanner.nextInt();
        horasTrabalhadas = scanner.nextInt();
        valorHoraTrab = scanner.nextDouble();

        /* processamento */
        salario = valorHoraTrab * horasTrabalhadas;

        /*saida */
        System.out.println("NUMBER = " + n);
        System.out.printf("SALARY = U$ %.2f\n", salario);

        scanner.close();
    }
}

File: 1-Iniciante/Ex1008.kt
import java.util.Scanner

class Funcionario(
    val numero: Int,
    val horasTrabalhadas: Int,
    val salarioPorHora: Double
) {

    var salario: Double = this.horasTrabalhadas * this.salarioPorHora
        private set
}

fun main(args: Array<String>) {
    val scanner = Scanner(System.`in`)
    val funcionario = Funcionario(
        numero = scanner.nextInt(),
        horasTrabalhadas = scanner.nextInt(),
        salarioPorHora = scanner.nextDouble()
    )

    println("NUMBER = ${funcionario.numero}")
    println("SALARY = U$ ${String.format("%.2f", funcionario.salario)}")
}
